types/stripe-js/payment-intents.d.ts

```diff
@@ -10,6 +10,7 @@ declare module '@stripe/stripe-js' {
     | CreatePaymentMethodEpsData
     | CreatePaymentMethodGiropayData
     | CreatePaymentMethodIdealData
+    | CreatePaymentMethodOxxoData
```

Comment:
Can you create an OXXO PaymentMethod without going through a PaymentIntent? e.g. `stripe.createPaymentMethod({type: 'oxxo', billing_details: {...}})`. The [reference](https://stripe.com/docs/js/payment_methods/create_payment_method) doesn't mention anything about this.